recipes/zeromq/zeromq.inc

```diff
@@ -0,0 +1,21 @@
+inherit autotools pkgconfig c++ auto-package-utils auto-package-libs
+
+RECIPE_TYPES = "machine sdk"
```

Comment:
Ah, you have a host tool for generating som crypto key.  That makes sense to have in sdk, but could then just as well be needed in native, to build keys directly in OE-lite.